sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Customizations/Models/MessageData.cs

```diff
@@ -17,7 +17,10 @@ public MessageData(int version, LogRecord logRecord) : base(version)
             Properties = new ChangeTrackingDictionary<string, string>();
             Measurements = new ChangeTrackingDictionary<string, double>();
             Message = LogsHelper.GetMessageAndSetProperties(logRecord, Properties).Truncate(SchemaConstants.MessageData_Message_MaxLength);
+#pragma warning disable CS0618 // Type or member is obsolete
```

Comment:
Is this document in the changelog of OpenTelemetry SDK?

Reply:
I found it in the 1.6.0-alpha changelog. We've never taken a dependency on a pre-release version of OpenTelemetry, so this is the first time we've been exposed to this change.

https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/src/OpenTelemetry/CHANGELOG.md#160-alpha1

Reply:
I still recommend removing this warning if it is not a blocker.

Reply:
Leave a TODO to remove this with stable release.